src/xopen/__init__.py

```diff
@@ -55,6 +54,9 @@
 # 128K buffer size also used by cat, pigz etc. It is faster than the 8K default.
 BUFFER_SIZE = max(io.DEFAULT_BUFFER_SIZE, 128 * 1024)
 
+# 1 is by far the fastest and most efficient level.
```

Comment:
I agree with your choice, but I’d prefer to avoid the hyperbole ("by far").

One could argue that "the fastest and most efficient level" would actually be 0 ... And level 9 would be the most efficient if you look at compression efficiency.

What I mean to say is: Choosing a compression level is an optimization problem with two criteria (speed and compression ratio). If you focus on one, you get the extremes ("no compression" and "infinite runtime"), so you need to choose a tradeoff that depends on the application. What we can do here in the library is to choose something sensible, but we shouldn’t call it "most efficient".

Can you just leave out the comment?

Reply:
That is a good idea. Unfortunately talking in hyperbole is one of my character flaws that is bound to pop up from time to time. Sorry about that. 

In this case I had switched the default around between 1, 5 and 6 a few times, because there are good reasons for all of these and the thought process got literary recoreded in the comment. I will remove it.

And that means yet another line less! I am quite happy with the refactoring so far.

Reply:
Done and removed.